plugins/ssl/ssl-certificate-expiry

```diff
@@ -11,7 +11,10 @@ ssl-certificate-expiry - Plugin to monitor Certificate expiration on multiple se
 =head1 CONFIGURATION
 
   [ssl-certificate-expiry]
-    env.services www.service.tld blah.example.net_PORT
+    env.services www.service.tld blah.example.net_PORT foo.example.net_PORT_STARTTLS
+
+PORT is the TCP port number
+STARTTLS is passed to openssl as "-starttls" argument. Useful for services like SMTP or IMAP implementing StartTLS.
```

Comment:
Please add an example here (e.g. "imap or ldap"). I guess, this would make the usage a bit more obvious.
(`STARTTLS` as a parameter name is quite unspecific - but I fail to imagine something better).

Reply:
An example is below.

Reply:
Yes, but just from my personal first attempt of reading this part, I failed to understand the meaning of the `STARTTLS` placeholder (since I do not know the manpage of `s_client` by heart). Just a suggestion, anyway ...